testbenches/cosimulation.mk

```diff
@@ -191,20 +191,30 @@ $(REGRESSION_TESTS:%=.%/vanilla_stats.csv): .%/vanilla_stats.csv : %.rule
 %.debug.log: %.vpd ;
 # Running simulation generates <test_name>/vcdplus.vpd. Move it to
 # <test_name>.vpd
-%.vpd: .%/vcdplus.vpd %.vanilla_stats.csv %.vcache_stats.csv
+
+# set the the name for MEMSYS_STATS
+ifneq ($(filter e_vcache_%, $(CL_MANYCORE_MEM_CFG)),)
+MEMSYS_STATS := vcache_stats
+endif
+
+ifneq ($(filter e_infinite_mem, $(CL_MANYCORE_MEM_CFG)),)
+MEMSYS_STATS := infinite_mem_stats
+endif
+
+%.vpd: .%/vcdplus.vpd %.vanilla_stats.csv %.$(MEMSYS_STATS).csv
```

Comment:
Thanks for doing this. Part of me wonders though, should we just unify on some generic name like `memory_system_stats`?

Reply:
that might imply that they have the same format - which I'm not sure is true.

Reply:
Good point

Reply:
@tommydcjung or @bornaehsani Do the different memory system files have the same format? Presumably no?